ansible_collections/arista/avd/docs/release-notes/4.x.x.md

```diff
@@ -18,3 +18,30 @@ Breaking changes may require modifications to the inventory or playbook. See the
 for details.
 
 <!-- list of breaking changes to come -->
+#### Change upper case CVP roles and module vars to lower case
+
+Potentially breaking in rare cases where custom logic relies on the registered vars or content of files.
+The custom scripts must be updated to use the lowercase variable instead.
+
+Component(s) name
+Roles:
+arista.avd.eos_config_deploy_cvp
+arista.avd.cvp_configlet_upload
+
+Modules:
+arista.avd.inventory_to_container
+arista.avd.configlet_build_config
```

Comment:
```suggestion
Modules:
* arista.avd.inventory_to_container
* arista.avd.configlet_build_config
```

Reply:
Implemented the fixes. I don't think we should have this in the porting guide, since we have no way of knowing how the vars are consumed by custom logic.